bin/pygrb/pycbc_pygrb_efficiency

```diff
@@ -124,8 +117,7 @@ if not opts.newsnr_threshold:
 outdir = os.path.split(os.path.abspath(opts.background_output_file))[0]
 trig_file = opts.offsource_file
 onsource_file = opts.onsource_file
-found_file = opts.found_file
-missed_file = opts.missed_file
```

Comment:
This will not work, will it? `missed file` is still being used in the next line.

Reply:
I cannot try the efficiency code with the new hdf5 output from `pycbc_multi_inspiral` yet (it doesn't contain all the necessary information).  But you are right, this way it is really broken :-). I changed the `missed_file` left to `found_missed_file` so this will fail in a more educated way when we try to use it on more complete hdf5 files.